rudder-web/src/main/scala/com/normation/rudder/web/components/RuleCompliance.scala

```diff
@@ -115,137 +113,177 @@ class RuleCompliance (
    * For each table : the subtable is contained in td : details
    * when + is clicked: it gets the content of td details then process it
    * as a datatable
+   *
+   * The table are empty when the page is displayed, then there is a JS call
+   * to refresh() that fill them.
    */
   def showCompliance : NodeSeq = {
+    <div id="directiveComplianceSection" class="unfoldedSection" onclick="$('#directiveCompliance').toggle(400); $('#directiveComplianceSection').toggleClass('foldedSection');$('#directiveComplianceSection').toggleClass('unfoldedSection');">
+      <div class="section-title">Compliance by Directive</div>
+    </div>
+    <div id="directiveCompliance">
+      <table id="reportsGrid" cellspacing="0">  </table>
+    </div>
+    <div id="nodeComplianceSection" class="unfoldedSection" onclick="$('#nodeCompliance').toggle(400); $('#nodeComplianceSection').toggleClass('foldedSection');$('#nodeComplianceSection').toggleClass('unfoldedSection');">
+      <div class="section-title">Compliance by Node</div>
+    </div>
+    <div id="nodeCompliance">
+       <table id="nodeReportsGrid" cellspacing="0">  </table>
+    </div>
+    <div id="recentChangesSection" class="unfoldedSection" onclick="$('#recentChanges').toggle(400); $('#recentChangesSection').toggleClass('foldedSection');$('#recentChangesSection').toggleClass('unfoldedSection');">
+      <div class="section-title">Recent changes</div>
+    </div>
+    <div id="recentChanges">
+      {SHtml.button("Refresh", () => refresh())}
```

Comment:
I believe it's better to stay as near as possible to the previous user experience. If the user used to click somewhere to refresh everythink, we can't change that in a minor release.